rollup.config.js

```diff
@@ -53,22 +54,7 @@ const mainConfig = Object.assign({}, baseConfig, {
 })
 
 const umdConfig = Object.assign({}, baseConfig, {
-  plugins: basePlugins.concat(
-    alias({
-      emotion: path.resolve(__dirname, './packages/emotion/src/index.js'),
-      'emotion-utils': path.resolve(
-        __dirname,
-        './packages/emotion-utils/src/index.js'
-      ),
-      'create-emotion': path.resolve(
-        __dirname,
-        './packages/create-emotion/src/index.js'
-      ),
-      'create-emotion-styled': path.resolve(
-        __dirname,
-        './packages/create-emotion-styled/src/index.js'
-      )
-    }),
+  plugins: [alias(lernaAliases())].concat(basePlugins).concat(
```

Comment:
alias has to come first otherwise if we build a single package and others have their built dist files ready `rollup-plugin-node-resolve` will kick in first to resolve the package and the dist files will be used

putting alias first we make sure that it always "wins" and source files will be used